dts/xtensa/intel/intel_adsp_cavs25.dtsi

```diff
@@ -260,11 +260,13 @@
 			status = "okay";
 		};
 
+#if 0		   /* FIXME what's the right address for this node? */
```

Comment:
IIRC, it is actually the same IP block. There are multiple "instances" or "channels" where they share the some common registers (e.g. control and status). That's why they have the same base address. They appear as two separate devices because these will be used by downstream drivers (referred using the `DT_*` macros).

Not sure what would be a good way to express this in devicetree.

Reply:
@mbolivar-nordic Thanks.

@abonislawski @juimonen Could you take a look and update the drivers and device trees?